fake-ppud-automation-api/__files/get-offender-X098092.json

```diff
@@ -32,6 +32,7 @@
           "indexOffence": "Blasphemy against coca cola",
           "dateOfIndexOffence": "2008-06-12"
         },
+        "releaseDate": "2005-05-12",
```

Comment:
Should this be `2013-02-02` as it is in the existing release? Or are you changing it deliberately? My only concern would be whether e2e tests rely on the date in some way for one of the test cases.

Reply:
Discussed: it did not affect the tests.